lib/src/Uncertainty/Bayesian/MetropolisHastingsImplementation.cxx

```diff
@@ -185,7 +188,7 @@ Point MetropolisHastingsImplementation::getRealization() const
     const Scalar newLogPosterior = computeLogPosterior(newState);
     
     // alpha = posterior(newstate)/posterior(oldstate)
-    const Scalar alphaLog = newLogPosterior - currentLogPosterior_;
+    const Scalar alphaLog = newLogPosterior - currentLogPosterior_ + probCurrentConditionedToNew_ - probNewConditionedToCurrent_;
```

Comment:
you've got to mention #1260 in the commit message (and not the changelog)

Reply:
We changed the commit message accordingly